docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -2,234 +2,205 @@
+[float]
+[[alerting-rest-api]]
+=== Using REST APIs
 
-For details on diagnosing the underlying causes of such delays, see <<task-manager-health-tasks-run-late>>.
+There is a fairly rich set of HTTP endpoints to introspect and manage rules and connectors.  
+In addition, there is a command-line client that uses legacy Rules and Connectors APIs, which be easier to to use, but it needs to be updated for the new APIs:
 
-Alerting and action tasks are identified by their type.
+https://github.com/pmuellr/kbn-action
+executing an action ad-hoc
+One of the http endpoints available for actions is the POST _execute API (reference).  You can use this to “test” an action.  For instance, if you have a server log action created, you can execute it via curling the endpoint:
+[source, txt]
+--------------------------------------------------
+curl -X POST -k \
+ -H 'kbn-xsrf: foo' \
+ -H 'content-type: application/json' \
+ https://elastic:changeme@localhost:5601/api/actions/connector/a692dc89-15b9-4a3c-9e47-9fb6872e49ce/_execute \
+ -d '{"params":{"subject":"hallo","message":"hallo!","to":["me@example.com"]}}'
+--------------------------------------------------
 
-* Alerting tasks always begin with `alerting:`. For example, the `alerting:.index-threshold` tasks back the <<rule-type-index-threshold, index threshold stack rule>>.
-* Action tasks always begin with `actions:`. For example, the `actions:.index` tasks back the <<index-action-type, index action>>.
+Another option is CLI tools to list / create / edit / delete alerts and actions are available in https://github.com/pmuellr/kbn-action[kbn-action]. Install via command:
+[source, txt]
+--------------------------------------------------
+    npm install -g pmuellr/kbn-action  
+--------------------------------------------------
 
-When diagnosing issues related to Alerting, focus on the tasks that begin with `alerting:` and `actions:`.
+and the same REST execute command will be:
+[source, txt]
+--------------------------------------------------
+kbn-action execute a692dc89-15b9-4a3c-9e47-9fb6872e49ce ‘{"params":{"subject":"hallo","message":"hallo!","to":["me@example.com"]}}’
+--------------------------------------------------
 
-For more details on monitoring and diagnosing task execution in Task Manager, see <<task-manager-health-monitoring>>.
+The result of this http request (and printed to stdout by https://github.com/pmuellr/kbn-action[kbn-action]) will be data returned by the action execution, along with error messages if errors were encountered.
 
 [float]
-[[connector-tls-settings]]
-=== Connectors have TLS errors when executing actions
-
-*Problem*:
+[[alerting-framework-health-ui]]
+=== Alerting Framework Health
```

Comment:
The heading doesn't match what's in this section.  Can it be "Look for error banners"